apis/v1alpha1/generator.yaml

```diff
@@ -97,6 +97,11 @@ resources:
       ignore: true
   EventSourceMapping:
     fields:
+      AmazonManagedKafkaEventSourceConfig:
```

Comment:
I think you meant `Queues` here :)

We still don't have an AMK controller to support resource references with/for EventSourceConfigs

Reply:
Thanks, I corrected it.